test/jibu/backup_test.go

```diff
@@ -75,23 +87,21 @@ var _ = Describe("use jibu api", func() {
 		BeforeEach(func() {
 			MyBy("clean up at the beginning")
 			_, _, _ = jibuClient.BackupPlanTagApi.DeleteBackupPlan(ctx, tenant, backupPlanName)
-			_, _, _ = jibuClient.BackupJobTagApi.DeleteBackupJob(ctx, tenant, backupJobName)
+			_ = deleteJobsOfBackupPlan(jibuClient, tenant, backupPlanName)
 			_, _, _ = jibuClient.RestorePlanTagApi.DeleteRestorePlan(ctx, tenant, restorePlanName)
 			_, _, _ = jibuClient.RestoreJobTagApi.DeleteRestoreJob(ctx, tenant, restoreJobName)
 		})
 
 		AfterEach(func() {
 			if cleanUpOnEnd {
 				MyBy("clean up at the end")
-				backupJob, _, err := jibuClient.BackupJobTagApi.GetBackupJob(ctx, tenant, backupJobName)
-				if err == nil {
-					MyBy(spew.Sdump("backupjob", backupJob))
-				}
+				_, _, _ = jibuClient.BackupPlanTagApi.DeleteBackupPlan(ctx, tenant, backupPlanName)
+				_ = deleteJobsOfBackupPlan(jibuClient, tenant, backupPlanName)
 				restoreJob, _, err := jibuClient.RestoreJobTagApi.GetRestoreJob(ctx, tenant, restoreJobName)
 				if err == nil {
 					MyBy(spew.Sdump("restorejob", restoreJob))
 				}
-				_, _, _ = jibuClient.BackupJobTagApi.DeleteBackupJob(ctx, tenant, backupJobName)
+				_, _, _ = jibuClient.RestorePlanTagApi.DeleteRestorePlan(ctx, tenant, restorePlanName)
```

Comment:
why delete restore plan? I think it has no harm to later testings.

Reply:
Just thought the plan is no longer needed. If this deletion is not wanted, I'll change it.

Reply:
the general idea is to keep some data/clues in case we want to debug after a long-run test. Deleting it is fine, but may need print it in log before delete it.

Reply:
Okay, I'll leave the restorePlan and delete the backupPlan(with a spew before) if it's a repeated one, since the latter will generate more backupjobs.